hera_sim/visibilities/healvis_wrapper.py

```diff
@@ -49,7 +69,7 @@ def sky_model(self):
 
         # convert from Jy/sr to K
         intensity = 10**-26 * self.sky_intensity.T
-        intensity *= (cnst.c.to("m/s").value/self.sky_freqs)**2 / (2 * cnst.k_B.value)
+        intensity *= old_div((old_div(cnst.c.to("m/s").value,self.sky_freqs))**2, (2 * cnst.k_B.value))
```

Comment:
What's the motivation for using old division here?

Reply:
I used the `futurize` command to automatically convert python 2 files to python 3 -- it must have automatically replaced every instance of division with `old_div`

Reply:
I can replace them back though, makes the code less clear

Reply:
Ah, gotcha--the `futurize` command thought that divisions were intended to be the old division. I think that we should only have `old_div` be used if there is a case where we absolutely want to be using it. As far as I know, there aren't any instances where we would prefer to use it, so it shouldn't be there.

Reply:
Removed all cases of old_div, I believe.

Reply:
All code in the repo has been written with py3 in mind (though it currently explicitly supports py2). So there shouldn't be _any_ `old_div`. Just make sure you do a "Search in path" on the repo to find all occurences.

Can I ask why you ran futurize?